docs/lib/examples/Card.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import { Card, CardImg, CardText, CardBody,
   CardTitle, CardSubtitle, Button } from 'reactstrap';
+import { getPlaceholderImg } from './util';
 
 const Example = (props) => {
   return (
     <div>
       <Card>
-        <CardImg top width="100%" src="https://placeholdit.imgix.net/~text?txtsize=33&txt=318%C3%97180&w=318&h=180" alt="Card image cap" />
+        <CardImg top width="100%" src={getPlaceholderImg({ w: 318, h: 180 })} alt="Card image cap" />
         <CardBody>
           <CardTitle>Card title</CardTitle>
           <CardSubtitle>Card subtitle</CardSubtitle>
           <CardText>Some quick example text to build on the card title and make up the bulk of the card's content.</CardText>
-          <Button>Button</Button>
+          <Button>Button</Button>03:26 PM - 
```

Comment:
```03:26 PM -  ``` what is this?

Reply:
🤐🤦sorry.. that was a typo. I was trying out an extension in VSCode. I've removed it in the updated code.